src/app/shared/ui-material/header/header.component.html

```diff
@@ -0,0 +1,14 @@
+<mat-toolbar color="primary">
+<a mat-button routerLink="">
+    <mat-icon class="computer-icon">computer</mat-icon>
+    <span class="app-name">
+    Gadget Store
```

Comment:
declare app title as a const in a component.
Use interpolation and titlecase pipe to insert it in a template.
in the template it must have h1 tag